saga-web/src/main/resources/saga-frontend/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
@anvithks We can remove this license now, this will now be added as a part of release/license

Reply:
@asifdxtreme This is fixed